readest: 0.9.30 -> 0.9.32

ddogfoodd opened the pull request:
Changelog: https://github.com/readest/readest/releases/tag/v0.9.32
Diff: https://github.com/readest/readest/compare/v0.9.30...v0.9.32
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ddogfoodd committed:
readest: 0.9.30 -> 0.9.31

Changelog: https://github.com/readest/readest/releases/tag/v0.9.31
Diff: https://github.com/readest/readest/compare/v0.9.30...v0.9.31

eljamm commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 397109`

Logs: https://github.com/eljamm/nixpkgs-review-gha/actions/runs/14339504431


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

eljamm reviewed (approved):
LGTM, Thanks!

ddogfoodd committed:
readest: 0.9.31 -> 0.9.32

Changelog: https://github.com/readest/readest/releases/tag/v0.9.32
Diff: https://github.com/readest/readest/compare/v0.9.31...v0.9.32

eljamm commented:
Waiting for [nixpkgs-review](https://github.com/eljamm/nixpkgs-review-gha/actions/runs/14351360019) to complete, which is taking some time on Linux because it's building `webkitgtk`.

By the way, @ddogfoodd, you usually don't need to mark the PR as draft before you push a new commit or rewrite history. When you'd probably need to do that is when changing branches (from master to staging, for example) so it doesn't mass-ping maintainers.

eljamm commented:
Ah, I see. I personally think a comment saying that you're updating the package is enough for maintainers to know not to merge, and for people subscribed to the PR to not be pinged too much. Even if we merged before updating in this case, it's not a big deal since the merged changes should be tested and good, and we can just make a follow-up PR.

eljamm commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 397109`

Logs: https://github.com/eljamm/nixpkgs-review-gha/actions/runs/14351360019


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>
